styles/style_guide/Periods.yml

```diff
@@ -0,0 +1,7 @@
+extends: existence
+message: "Don't use periods with acronyms or initialisms such as '%s'."
```

Comment:
Not sure about this rule; would it create a false positive for `e.g.` and `i.e.`? If I remember correctly, we are still undecided on these.

Maybe I'm wrong and it only works on capitalized acronyms.